docs/spark/tutorials/get-started-macos-instructions.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Get started with .NET for Apache Spark on macOS
+description: Discover how to run a .NET for Apache Spark app using .NET Core on MacOSX.
+ms.date: 01/22/2019
+ms.topic: tutorial
+ms.custom: mvc
+# Customer intent: As a developer, I want to write a simple custom application using .NET for Apache Spark on MacOSX.
+---
+
+# Tutorial: Get started with .NET for Apache Spark on macOS
+
+This tutorial shows you how to run a .NET for Apache Spark app using .NET Core on macOS.
+
+> [!div class="checklist"]
+>
+> * Prerequisites to run a .NET for Apache Spark application
+> * Write a simple .NET for Apache Spark application
+> * Build and run a .NET for Apache Spark application
+
+## Prerequisites
+
+- Download and install **[.NET Core 3.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1)** 
+- Install **[Java 8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)** 
+  - Select the appropriate version for your operating system, for example, `jdk-8u231-macosx-x64.dmg`.
+  - Install using the installer and verify you are able to run `java` from the command line
+- Download and install **[Apache Spark 2.4.4](https://archive.apache.org/dist/spark/spark-2.4.4/spark-2.4.4-bin-hadoop2.7.tgz)**:
+  - Add the necessary environment variable SPARK_HOME with a value of, for example, `~/bin/spark-2.4.4-bin-hadoop2.7/`
+  
+    ```bash
+    export SPARK_HOME=~/bin/spark-2.4.4-bin-hadoop2.7/
+    export PATH="$SPARK_HOME/bin:$PATH"
+    source ~/.bashrc
+    ```
+    
+- Download and install **[Microsoft.Spark.Worker](https://github.com/dotnet/spark/releases)** release:
+  - Select a **[Microsoft.Spark.Worker](https://github.com/dotnet/spark/releases)** release from .NET for Apache Spark GitHub Releases page and download into your local machine (for example, `/bin/Microsoft.Spark.Worker/`).
```

Comment:
Missing "~" ?
Correct me if it I'm wrong.

```suggestion
  - Select a **[Microsoft.Spark.Worker](https://github.com/dotnet/spark/releases)** release from .NET for Apache Spark GitHub Releases page and download into your local machine (for example, *~/bin/Microsoft.Spark.Worker/*).
```

Reply:
You are right, good catch! Thanks.